esp_jpeg/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "1.3.0"
+version: "1.3.1"
```

Comment:
Could you please add an entry in https://github.com/espressif/idf-extra-components/blob/55c79124f1973c565cae1683a4274ed87dc8ac6a/esp_jpeg/CHANGELOG.md?